docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1402,3 +1402,58 @@ Test awesome shareable disks150+0 records in
 ```
 
 If you are using local devices or RWO PVCs, setting the affinity on the VMs that share the storage guarantees they will be scheduled on the same node. In the example, we set the affinity on the second VM using the label used on the first VM. If you are using shared storage with RWX PVCs, then the affinity rule is not necessary as the storage can be attached simultaneously on multiple nodes.
+
+## File Systems
+
+Currently, only `virtIO-FS` file system is supported.
+### virtIO-FS
+
+`virtIO-FS` makes a Persistent Volume Claim visible in the `kubevirt` VM.
+
+> **Note:** The **ExperimentalVirtiofsSupport** feature gate
+> [must be enabled](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
+> to use this volume. Available starting with KubeVirt v0.36.0.
+
+Example:
+
+    metadata:
+      name: testvmi-fs
+    apiVersion: kubevirt.io/v1
+    kind: VirtualMachine
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          filesystems:
+          - name: virtiofs-disk
+            virtiofs: {}
+        resources:
+          requests:
+            memory: 128Mi
+      volumes:
+        - name: containerdisk
+          containerDisk:
+            image: quay.io/containerdisks/fedora:latest
+        - name: virtiofs-disk
+          persistentVolumeClaim:
+            claimName: mypvc
+
```

Comment:
Add a line of its own with three backticks.

Reply:
Fixed